perf/README.md

```diff
@@ -10,21 +10,27 @@ Benchmark results can be visualized with https://observablehq.com/@mxinden-works
 
 ## Provision infrastructure
 
-1. `cd terraform`
-2. Save your public SSH key as the file `./user.pub`.
+### Bootstrap
+
+1. Save your public SSH key as the file `./regions/files/user.pub`; or generate a new key pair with `make ssh-keygen` and add it to your SSH agent with `make ssh-add`.
+2. `cd terraform`
 3. `terraform init`
 4. `terraform apply`
+5. `CLIENT_IP=$(terraform output -raw client_ip)`
+6. `SERVER_IP=$(terraform output -raw server_ip)`
```

Comment:
> It adds an additional step to testing against a personal AWS account. (Spin up both long-lived and short-lived resources.)

I restored the original instructions here. With personal AWS account, `terraform apply` will create all the resources required for running the test here, including nodes.